integration_test/sql/logging.exs

```diff
@@ -61,14 +61,38 @@ defmodule Ecto.Integration.LoggingTest do
              TestRepo.insert!(%Post{title: "1"}, [log: :error])
            end) =~ "[error]"
 
-    # We cannot assert on the result because it depends on the suite log level
-    ExUnit.CaptureLog.capture_log(fn ->
-      TestRepo.insert!(%Post{title: "1"}, [log: true])
-    end)
+    before_log_level = Logger.level()
+
+    try do
+      Logger.configure(level: :debug)
+      assert ExUnit.CaptureLog.capture_log(fn ->
+        TestRepo.insert!(%Post{title: "1"}, [log: true])
+      end) =~ "[debug]"
+
+      refute ExUnit.CaptureLog.capture_log(fn ->
+        TestRepo.insert!(%Post{title: "1"}, [log: true])
+      end) =~ " ↳ "
+    after
+      Logger.configure(level: before_log_level)
+    end
 
```

Comment:
Not sure if this really has an impact, but logger level is for the whole application, so changing it through `Log.configure` may affect other tests (as this test is async)

Reply:
@v0idpwn 
Thanks for review！
I understood. But I don't know how to test it other than that.
Please give me a hint🙇‍♂️